src/parking/controls/editor/editor-form.ts

```diff
@@ -167,7 +167,7 @@ function getTagInput(osm: OsmWay, side: string, parkingType: string, tagTemplate
         }
         case 'parking:lane:{side}:surface': {
             input = getTextInput(tag, value)
-            input.placeholder = osm.tags.surface ?? input.placeholder
+            input.placeholder = `eg. ${osm.tags.surface ?? input.placeholder}`
```

Comment:
```suggestion
            input.placeholder = `e.g. ${osm.tags.surface ?? input.placeholder}`
```

See https://en.wiktionary.org/wiki/e.g.